src/components/QuranReader/VerseReader/index.tsx

```diff
@@ -0,0 +1,85 @@
+import styled from 'styled-components';
```

Comment:
We should reuse the same component for the chapters routes. 

The api request is different, otherwise we should treat them as identical.

The same applies for the Reader component